resources/js/d3.visualization.js

```diff
@@ -33,7 +33,7 @@
 
     var graticule = d3.geo.graticule();
 
-    var svg = d3.select("#content").append("div").attr("class","map-wrapper").append("svg")
+    var svg = d3.select(".map-wrapper").append("svg")
```

Comment:
This is the one javascript change I mentioned in the PR.

Reply:
Ha, you can totally tell when I converted jQuery to D3. Good call.